### ISSUE TITLE
Scheduler is updated once a rollout while optimizer is updated once a step in PPO

### ISSUE THREAD

**Issue by skpig** [volcengine/verl#399]
As shown in:

https://github.com/volcengine/verl/blob/2440aa6993b32ec479691960226d9bfa36d07a04/verl/workers/fsdp_workers.py#L420-L430

scheduler is updated outside ` self.actor.update_policy(data=data)`, in which the optimizer is updated for multiple steps.

I'm wondering if this is the intended or correct implementation.

**Comment by skpig** [comment]
A related question is that:
what exactly does the term "training_step" mean in verl? 
Could you please explain it?

**Comment by PeterSH6** [comment]
The `training_step` in verl means one `rollout generation` time.
With the response and experiences generated from one `rollout`, we can update the policy multiple times. The number of updating (optimizer.step()) time is computed by `train_batch_size // ppo_mini_batch_size`.

**Comment by skpig** [comment]
Thanks for your reply!
